DependencyInjection/HttplugExtension.php

```diff
@@ -152,6 +156,9 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(0, new Reference($config['journal']));
                 break;
             case 'logger':
+                if (class_exists('Http\Client\Common\Plugin\LoggerPlugin')) {
```

Comment:
I thought we support PHP 5.4, but just checked and we don't, so this is a good idea, thanks.